tests/jeos/prepare_rpi_image.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
```

Comment:
Only `Copyright © 2018 SUSE LLC` since it is new module

Reply:
solved

Reply:
Also please remove `Copyright © 2009-2013 Bernhard M. Wiedemann` this line as well, sorry for not mention it in my first comment :)

Reply:
np, fixed